packages/cfa-template/template/backend/serverless.yml

```diff
@@ -10,24 +11,26 @@ provider:
     stage: ${opt:stage}
     # Change for your desired environment
     # https://docs.aws.amazon.com/systems-manager/latest/userguide/ps-integration-lambda-extensions.html#ps-integration-lambda-extensions-add
-    layers:
-        - arn:aws:lambda:<AWS_REGION>:<AWS_ACCOUNT_ID>:layer:AWS-Parameters-and-Secrets-Lambda-Extension:<VERSION>
+    # vv Uncomment for leveraging AWS Secrets Manager
```

Comment:
Maybe in the future we could have a step when you run `create-frigg-app` where you choose if you want to use the AWS Secrets Manager. And then based on that we add or remove those pieces of code.

Reply:
Agreed- really, we'll want to prompt someone to make all sorts of decisions:
- Cloud provider: AWS | GCP | Azure | Docker | other
- Secrets Management: (Provider dependent)
- Database Source: BYO | Provision for me
- Database Type: MongoDB | Postgres | MySQL ... cloud provider options
- Encryption Key Management: ENVs (AES Key with manual rotation) | ...Cloud provider options